src/hapi/pipelines/configs/core.yaml

```diff
@@ -38,3 +38,111 @@ org_type:
   resource: "Organization Types (Beta) - CSV"
   format: "csv"
   headers: 2
+
+sector_map:
```

Comment:
David also suggested turning these into google sheets that the scraper/loader reads from, so that's a possibility!  I think the config would be more straightforward, but the google sheet would allow say DPT to edit or add items to the list.

Reply:
For now I shortened the config to match only what's in the three countries we're using, so it's much more manageable.